src/openms/source/ANALYSIS/TOPDOWN/FLASHDeconvAlgorithm.cpp

```diff
@@ -346,8 +346,8 @@ namespace OpenMS
     long binEnd = (long) massBins.size();
     auto candidateMassBinsForThisSpectrum = boost::dynamic_bitset<>(massBins.size());
     // how many peaks of continuous charges per mass
-    int *continuousChargePeakPairCount = new int[massBins.size()];
-    std::fill_n(continuousChargePeakPairCount, massBins.size(), 0);
+    int *supportPeakPairCount = new int[massBins.size()];
```

Comment:
Dynamic arrays
Using a pointer to a malloc-ed/new-ed array will be at best as fast as the std::vector version, and a lot less safe (see litb's post). So better use a std::vector.

Reply:
I will change them all then! So just vector<...>(numberofelements, initval) is the way to go? Thanks!